test/e2e/framework/events/events.go

```diff
@@ -35,11 +35,12 @@ import (
 	"github.com/onsi/ginkgo"
```

Comment:
The dependencies seem clean and good for me for moving the package into e2e framework.